README.md

```diff
@@ -1,5 +1,45 @@
-First run ```python all.py```.
+# Guided transfer learning
+This repository implements our Guide transfer learning approach (see [our article](https://www.researchgate.net/publication/367378102_Guided_transfer_learning_Helping_deep_networks_when_learning_gets_tough)).
 
-Then run ```tensorboard --logdir logs --bind_all```
+# Installation
+1. Clone the repository
+```bash
+git clone https://github.com/RobotsGoMental/gtl_poc.git
+cd gtl_poc
+```
+2. Create a new conda environment with Python 3.9
+```bash
+conda create --name gtl python=3.9
```

Comment:
I wouldn't use "conda" but rather "venv" which is more vanilla, less opinionated. "conda" is very datasciency, so maybe it makes sense to move it completely, if those are our customers.

Personally I would rather use pipenv, poetry and venv before going for conda.

Reply:
Let's go with venv as @vjekoslav proposed it's less opinionated.